Gopkg.toml

```diff
@@ -97,7 +97,7 @@ required = [
 
 [[constraint]]
   name = "github.com/spf13/viper"
-  version = "^1.0.0"
+  version = "^1.5.0"
```

Comment:
Is this change necessary? If we're changing this file, we need to run `dep ensure --update` and also check-in the lock file.

Reply:
It is - there was a bug in Viper where `isSet` was returning true even in cases where it was not set, but there was a default value. This was fixed in 1.5